content/docs/glossary/phishing.md

```diff
@@ -13,5 +13,3 @@ navigation:
 Phishing is a technique for acquiring information such as user names, passwords, credit cards, social security numbers or other personal data by masquerading as a trusted entity like a bank or credit card company. Phishing emails appear to be sent by the trusted entity, which can cause the consumer to be tricked into providing their personal information.
 
 SendGrid employs both technology and staff to prevent the sending of phishing emails. We have a solid track record of working with the global email community to detect and stop the sending of phishing and spam emails.
-
-To get more information please check out our [Email Infrastructure Guide](https://go.sendgrid.com/SendGrid-Infrastructure-Guide.html?mc=Direct&mcd=https://sendgrid.com/docs/index.html).
```

Comment:
Thanks for bringing this to our attention. I believe we want to replace the link with this url: https://sendgrid.com/resource/the-email-infrastructure-guide-build-it-or-buy-it/

Would you mind updating the PR with that link before we merge it in? Thanks again for your work.

Reply:
Correct link added back in.